docs/source/examples/checks/methodology/single_feature_contribution_train_test.ipynb

```diff
@@ -5,123 +5,83 @@
    "id": "21c5d5bc",
```

Comment:
<p>I think this is fine and we don't need to show here the "unmodified" result, but what the rest of you think? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I agree, this is fine</p>